FastTimeSeries/src/models/RNN.jl

```diff
@@ -1,3 +1,9 @@
+function tabular2rnn(X::AbstractArray{Float32, 3})
+    X = permutedims(X, (2, 1, 3))
+    X = [X[t, :, :] for t ∈ 1:size(X, 1)]
```

Comment:
```suggestion
    X = collect(eachslice(X; dims=1))
```
Should be more efficient. But see comments below.